plugins/woocommerce-beta-tester/src/tools/data/actions.js

```diff
@@ -210,3 +210,22 @@ export function* runDisableEmail() {
 		yield setIsEmailDisabled( response );
 	} );
 }
+
+export function* resetCustomizeYourStore() {
+	yield runCommand( 'Reset Customize Your Store', function* () {
+		const optionsToDelete = [
+			'woocommerce_customize_store_onboarding_tour_hidden',
+			'woocommerce_admin_customize_store_completed',
+			'wc_blocks_patterns_content',
```

Comment:
```suggestion
			'woocommerce_admin_customize_store_completed_theme_id',
			'wc_blocks_patterns_content',
```

Reply:
I think we might as well include this option, even though it doesn't have any effect on its own currently. Someone might change it down the road and break this implicit assumption 🤷‍♂️